### DISCUSSION TITLE
Heatit Z-relay

### DISCUSSION THREAD

**Discussion by espenthingstad** [zwave-js/zwave-js#4402] [Q&A] [answered]
Hi! I have used the Zwave JS integration for Home Assistant, and has just added a Heatit Z-relay : 
https://github.com/zwave-js/node-zwave-js/blob/543e8608554c223d088315279aceaaa22bf7899a/packages/config/config/devices/0x019b/heatit_z-relay.json

I can control the relay, but I am not getting any information on the "sensors". I am told by vendor that it is due to the configuration in Zwave JS integration. What is needed to populate also values on the "sensors"? :)

I am also using Heatit ZTRM-3 in my system, with this config file, and the "sensors" for this device works properly. 
https://github.com/zwave-js/node-zwave-js/blob/543e8608554c223d088315279aceaaa22bf7899a/packages/config/config/devices/0x019b/z-trm3.json

**Picture of Heatit ZTRM-3 integration:**
![Heatit ZTRM-3](https://user-images.githubusercontent.com/80642192/160114166-bb4d0235-bb7c-4a7b-a0d4-1a44f3d89260.png)

**Picture of Heatit Z-relay integration:**
![Heatit Z-relay 25A](https://user-images.githubusercontent.com/80642192/160114160-b300b723-e50d-43fb-81a9-52ded5e0cb5e.png)

**Comment by AlCalzone** [comment]
Please make a driver log, loglevel "debug", re-interview the relay and post the log here as a file.

**Reply by espenthingstad** [reply]
Hi, see attached file. Not sure if interview finished, 15 minutes waiting, dialogue did not close.. 

It is device 007.
[zwave_js (1).log](https://github.com/zwave-js/node-zwave-js/files/8353362/zwave_js.1.log)

**Reply by AlCalzone** [reply]
There's almost nothing in that log. When an interview happens, you should see hundreds or thousands of log lines.

**Reply by espenthingstad** [reply]
Yes it seems as something happened in the interview phase. I did it again, now also with the "Device interview complete" notification. Hopefullt these logs are better, attached :) Still device 007.
[zwave_js (2).log](https://github.com/zwave-js/node-zwave-js/files/8357818/zwave_js.2.log)

**Comment by AlCalzone** [comment, marked as answer]
This is a bug. Lets continue discussion in https://github.com/zwave-js/node-zwave-js/issues/4413
